src/chapters/exceptions/exceptions-control-flow.rst

```diff
@@ -0,0 +1,72 @@
+Exceptions-Control-Flow
+=======================
+
+In some instances we receive input from a user, and we can't always ensure they will type in something that will run perfectly with our code.
+
+JavaScript gives us some control flow tools to handle exceptions, by catching thrown exceptions.
+
+When we are anticipating an exception, and we want our program to continue running, and not experience the runtime error we can tell JavaScript to **try** to run a block of code, and if a specific exception is thrown to **catch** the exception and to run a different block of code.
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+      :caption: Code
+
+      try {
+          console.log(x[0]);
+      }
+      catch(ReferenceError) {
+          console.log("We caught a ReferenceError, but our program continues to run!");
+          console.log("0");
+      }
+
+   .. sourcecode:: js
+      :caption: Output
+
+      We caught a ReferneceError, but our program continues to run!
+      0
+
+   In this example we attept to print out the first element of our variable x. We forgot to define x, but nested our code inside of a *try* statement, and specifically **catch** ReferenceErrors. It should be noted that only ReferenceErrors will be caught by the catch statement. However, we can define as many catch statements as we want.
```

Comment:
In this example, we attempt to print ... variable ``x``."

try is not bolded in the second sentence. Also, the comma before "and" is unnecessary. 

``ReferenceError``

Reply:
fixed